src/cascadia/TerminalSettingsEditor/Profiles.cpp

```diff
@@ -6,16 +6,22 @@
 #include "Profiles.g.cpp"
 #include "EnumEntry.h"
 
+#include <LibraryResources.h>
+
+using namespace winrt::Windows::UI::Text;
 using namespace winrt::Windows::UI::Xaml;
 using namespace winrt::Windows::UI::Xaml::Navigation;
 using namespace winrt::Windows::Foundation;
+using namespace winrt::Windows::Foundation::Collections;
 using namespace winrt::Windows::Storage;
 using namespace winrt::Windows::Storage::AccessCache;
 using namespace winrt::Windows::Storage::Pickers;
 using namespace winrt::Microsoft::Terminal::Settings::Model;
 
 namespace winrt::Microsoft::Terminal::Settings::Editor::implementation
 {
+    static const auto CustomFontWeight{ winrt::make<EnumEntry>(RS_(L"Profile_FontWeightCustom/Content"), winrt::box_value<uint16_t>(0u)) };
```

Comment:
_please_ stop allocating static storage for complex types

you can't do this for the same reason you can't make a static hstring.